s3fs/tests/test_s3fs.py

```diff
@@ -152,10 +152,6 @@ def test_info(s3):
     s3.touch(a)
     s3.touch(b)
     assert s3.info(a) == s3.ls(a, detail=True)[0]
-    parent = a.rsplit('/', 1)[0]
-    s3.dirs[parent].pop(0)  # disappear our file!
```

Comment:
Should still be there, because of the `ls`, above - no?

Reply:
What the ls adds to cache is the full path of a, it's actually the old implementation of info that adds it's parent to the cache, so no